Dockerfile

```diff
@@ -104,16 +104,16 @@ COPY . .
 COPY --from=deps /workspace-install ./
 
 # Optional: if the app depends on global /static shared assets like images, locales...
-RUN yarn workspace $PROJECT share-static-hardlink && yarn workspace $NEXT_PUBLIC_CMS_GRAPHQL_ENDPOINT build
+RUN yarn workspace $PROJECT share-static-hardlink && yarn workspace $PROJECT build
 
 # Does not play well with buildkit on CI
 # https://github.com/moby/buildkit/issues/1673
-RUN --mount=type=cache,target=/root/.yarn3-cache,id=yarn3-cache \
-    SKIP_POSTINSTALL=1 \
-    YARN_CACHE_FOLDER=/root/.yarn3-cache \
-    yarn workspaces focus $PROJECT --production
+# RUN --mount=type=cache,target=/root/.yarn3-cache,id=yarn3-cache \
+#     SKIP_POSTINSTALL=1 \
+#     YARN_CACHE_FOLDER=/root/.yarn3-cache \
+#     yarn workspaces focus $PROJECT --production
 
-CMD ["sh", "-c", "echo ${NEXT_PUBLIC_CMS_GRAPHQL_ENDPOINT}"]
```

Comment:
The CMD is unnecessary. I used it to debug with echo command. The original version of the [dockerfile](https://github.com/belgattitude/nextjs-monorepo-example/blob/main/Dockerfile#L66-L67) doesnt have it.

Reply:
there are more fixes in the #20 related to the dockerfile and deployment

Reply:
...I'm about to merge it here